CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 5.0.0 / 2023-05-17
+
+**Full Changelog**: https://github.com/opentensor/bittensor/compare/v4.0.1...v5.0.0
+
+
+## 4.1.0 / 2023-05-17
```

Comment:
I dont believe that this version was ever released, can we remove this from the changelog?

Reply:
Yep. Done.